log-insights/setup/opentelemetry.mdx

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
TODO: add link from https://pganalyze-we-add-selfho-wbo8ie.herokuapp.com/docs/log-insights/setup/self-managed/01_determine_log_location